server.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const express = require('express');
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.static('./public'));
+
+app.post('/articles', function(request, response) {
+  response.send('Recorded to server');
+})
+
+app.listen(PORT, function() {
```

Comment:
You need to put that console.log into this function. It is part of how we set up lab.